int64 x
int64 y
---
bool suc
